app/assets/stylesheets/govuk_publishing_components/components/_date-input.scss

```diff
@@ -0,0 +1,13 @@
+@import "govuk-frontend/components/date-input/date-input";
+
+.govuk-date-input {
```

Comment:
Worth a comment explaining why you're overwriting the upstream component?

Reply:
Yep, will add. I'm not happy that I'm overwriting using the original classes, but there's not way to pass classes in this lib, so I don't see another way.